package.json

```diff
@@ -15,7 +15,7 @@
     "coverage:report": "nyc report --reporter=lcov",
     "test": "./test/script/run_tests.sh",
     "test:cli": "node test/script/cli.js",
-    "test:deps": "dependency-check *.js 'lib/**/*.js' 'test/**/*.js' --no-dev -i async_hooks -i perf_hooks -i parseurl",
+    "test:deps": "dependency-check start.js index.js 'lib/**/*.js' 'test/**/*.js' --no-dev -i async_hooks -i perf_hooks -i parseurl",
     "test:tav": "tav --quiet",
```

Comment:
Reviewer note: I like to have "foo.js" and "play.js" files (etc.) in my working copy. It is unhelpful when those break `npm test` because they use some dev/test dependency, e.g.:

```
% npx dependency-check *.js 'lib/**/*.js' 'test/**/*.js' --no-dev -i async_hooks -i perf_hooks -i parseurl
Fail! Dependencies not listed in package.json: blocked-at
```

where "blocked-at" is only being used here for dev/testing (it was installed via `npm install --no-save blocked-at`.


I could move to a separate PR if you prefer.